browser/about.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>About</title>
+    <link rel="stylesheet" href="app.css">
+  </head>
+  <body>
+      <img class="image" style="margin-right:90px;margin-bottom:27px;" width="220px" src="images/RHDP_logo_RGB_default.svg"/>
+      <section class="welcome-panel">
+        <p>For help, please visit</p>
+        <ul>
+          <li><a href="https://access.redhat.com/documentation/en/red-hat-development-suite/">Documentation</a>.</li>
```

Comment:
These links loads referenced documents right in about window like this:

![image](https://user-images.githubusercontent.com/620330/29594770-11d8966a-8768-11e7-96da-b1785b10906e.png)